ext/native-decls/GetVehicleOilLevel.md

```diff
@@ -5,6 +5,7 @@ apiset: client
 ## GET_VEHICLE_OIL_LEVEL
 
 ```c
+// 0xFC7F8EF4
```

Comment:
Why does this have to be there?

Reply:
If I take a look in https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md I replace the information from:
```c
// 0x0123456789ABCDEF 0x1234ABCD
BOOL _EXAMPLE_NATIVE(int param);
```
so I replace by
```c
// 0xFC7F8EF4
float GET_VEHICLE_OIL_LEVEL(Vehicle vehicle);
```
If not necessary, I would make the change to remove it and take note of it in the future.